tests/test_validation.py

```diff
@@ -51,6 +51,20 @@ def test_id_auth(here: Path) -> None:
     assert list(errors) == []
 
 
+def test_id_type(here: Path) -> None:
+    dataset = dataset_schema_from_path(here / "files" / "id_type.json")
```

Comment:
I cannot find this `id_type.json`, is it check in into the repo?

Reply:
Of course it wasn't, but it is now.